### DISCUSSION TITLE
Any plan of support SSE (Server Sent Event)?

### DISCUSSION THREAD

**Discussion by LeviEyal** [mswjs/msw#2116] [Ideas]
Sorry if this question was already questioned before. couldn't find something related

**Comment by kettanaito** [comment]
Hi, @LeviEyal. That's a good question.

Some of our users have reported that you can already mock SSE with MSW. Those are still HTTP requests, and as long as you model their responses as they should be (mostly `content-type`), your application will receive the events. 

I've also created the #2117 proposal to have a first-class API for SSE in the future. There are no plans for working on that API right now. If that sounds interesting to you, please dive in! I will help with code reviews and seeing your contributions land for everyone. Thanks.
